build.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
```

Comment:
I don't think this file is necessary? Seems to be for yourself to build a ZIP which you can then install into Kodi?

Personally I'm must developing (/cloned the repository) into the Kodi addon directory so I don't need to build it and it allows for a much faster development cycle.

So could you please remove it?

Reply:
yeah, that was mainly so i could quickly build it, as i was then using `adb` to push it to a android devie Ill remove it from the branch